server/sonar-ce/src/test/java/org/sonar/ce/taskprocessor/CeWorkerFactoryImplTest.java

```diff
@@ -22,7 +22,9 @@
 
 import java.util.HashSet;
 import java.util.Set;
+import org.junit.Rule;
```

Comment:
I'll fix this, probably I started another test in the wrong class and I missed to remove the imports.